MdePkg/MdePkg.dec

```diff
@@ -2307,17 +2314,27 @@
   gEfiMdePkgTokenSpaceGuid.PcdSpinLockTimeout|10000000|UINT32|0x00000004
 
   ## The mask is used to control DebugLib behavior.<BR><BR>
-  #  BIT0 - Enable Debug Assert.<BR>
-  #  BIT1 - Enable Debug Print.<BR>
-  #  BIT2 - Enable Debug Code.<BR>
-  #  BIT3 - Enable Clear Memory.<BR>
-  #  BIT4 - Enable BreakPoint as ASSERT.<BR>
-  #  BIT5 - Enable DeadLoop as ASSERT.<BR>
+  #  BIT0 - Enable the ASSERT*** and CR macros.<BR>
+  #  BIT1 - Enable the DEBUG macro (does not affect the DEBUG_PRINT macro).<BR>
+  #  BIT2 - Enable the DEBUG_CODE_BEGIN macro.<BR>
+  #  BIT3 - Enable the DEBUG_CLEAR_MEMORY macro.<BR>
+  #  BIT4 - Enable BreakPoint for failed ASSERT.<BR>
+  #  BIT5 - Enable DeadLoop for failed ASSERT.<BR>
   # @Prompt Debug Property.
   # @Expression  0x80000002 | (gEfiMdePkgTokenSpaceGuid.PcdDebugPropertyMask & 0xC0) == 0
   gEfiMdePkgTokenSpaceGuid.PcdDebugPropertyMask|0|UINT8|0x00000005
 
-  ## This flag is used to control the print out Debug message.<BR><BR>
+  ## This flag is used to filter the output of the DEBUG_PRINT macro and DebugPrint functions.<BR>
+  #  The default value is 0x80000000, meaning that DebugPrint returns immediately for anything
+  #  except errors. You might set this to a value like 0x80000002 to also enable warnings.<BR><BR>
+  #  This flag is consumed by the BaseDebugPrintErrorLevelLib implementation of
+  #  GetDebugPrintErrorLevel(). The DebugPrint(Level, ...) functions return immediately if
+  #  0 == (GetDebugPrintErrorLevel() & Level). Other implementations of DebugPrintErrorLevelLib

```

Comment:
We prefer to avoid Yoda expressions. Would be better as `(GetDebugPrintErrorLevel() & Level) == 0`
Also, should we mention how `MDEPKG_NDEBUG affects debug prints?

Also, should we use the EDK2 coding style of a space between the function name and opening parenthesis? e.g. `GetDebugPrintErrorLevel ()`.